server/lib/src/valueset/session.rs

```diff
@@ -347,7 +347,7 @@ impl ValueSetT for ValueSetSession {
         Box::new(
             self.map
                 .iter()
-                .map(|(u, m)| format!("{}: {:?}", uuid_to_proto_string(*u), m)),
+                .map(|(u, m)| format!("Auth session id: {}, {:?}", uuid_to_proto_string(*u), m)),
```

Comment:
We can't "format" things in "to proto string". This is meant to be a stable transmitted representation of the data, not a human/user friendly display. So this needs to be undone sorry :)